package.json

```diff
@@ -11,6 +11,10 @@
         "url": "https://github.com/sponsors/ljharb"
     },
     "main": "lib/index.js",
+    "files":[
+        "dist",
+        "lib"
+    ],
```

Comment:
The `files` field is dangerous; it means needed files could be accidentally omitted. The only danger of `.npmignore` is that an unneeded file is unnecessarily included.

Reply:
Sorry, missed the npm ignore file.

Reply:
np, unfortunately npm's own docs recommend "files", but it's the more dangerous approach.